en/reference/annotations-mapping.rst

```diff
@@ -50,7 +50,9 @@ Optional attributes:
 
 -  **nodeType**: PHPCR type for this node, default ``nt:unstructured``.
 -  **repositoryClass**: Name of the repository to use for this document.
--  **versionable**: Set to true to enable versioning, implies *referenceable*.
+-  **versionable**: Set to ``"simple"`` or ``"full"`` to enable versioning 
+   (respectively simple or full level). Set to ``"false"`` to disable versioning
```

Comment:
So, what about:
**versionable**: (string) Set to ``simple`` or ``full`` to enable versioning (respectively simple or full level), ``false`` to disable versioning inheritance. Implies *referenceable*.

I think that using quotes around "false" and the other two values is just for clarity. I don't see the point to define a syntax rule for attributes description, because this is already done in data mapping files.